app/vmui/packages/vmui/src/pages/CustomPanel/InstantQueryTip/InstantQueryTip.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from "preact/compat";
+import Hyperlink from "../../../components/Main/Hyperlink/Hyperlink";
+
+const last_over_time = <Hyperlink
+  text="last_over_time"
+  href="https://docs.victoriametrics.com/MetricsQL.html#last_over_time"
+  underlined
+/>;
+
+const instant_query = <Hyperlink
+  text="instant query"
+  href="https://docs.victoriametrics.com/MetricsQL.html#last_over_time"
```

Comment:
It looks like this link is incorrect. It should point to https://docs.victoriametrics.com/keyConcepts.html#instant-query